app/src/main/java/agency/five/tmdb/navigation/Navigation.kt

```diff
@@ -0,0 +1,33 @@
+package agency.five.tmdb.navigation
+
+import agency.five.tmdb.DetailsScreen
+import agency.five.tmdb.MainScreen
+import agency.five.tmdb.MoviePerson
+import agency.five.tmdb.R
+import agency.five.tmdb.ui.theme.MovieItemViewState
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+
+@Composable
+fun Navigation() {
```

Comment:
Nice usage of Jetpack Compose navigation. Good job implementing the navigation in a way that Details screen is open "above" Main screen.